src/myPage/LearningPathDropdown.js

```diff
@@ -6,58 +6,96 @@
  *
  */
 
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import polyglot from '../i18n';
 import Icon from '../common/Icon';
 import { learningPathStatuses } from '../util/learningPathStatuses';
 
-export function LearningPathDropdown({ onSelect, learningPath }) {
-  const makeOnClick = actionType => (evt) => {
+export default class LearningPathDropdown extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      displayDropDown: false,
+    };
+  }
+
+  makeOnClick = actionType => (evt) => {
+    const { learningPath, onSelect } = this.props;
     evt.preventDefault();
     onSelect(actionType, learningPath);
   };
 
 
-  const publishAction = status => (evt) => {
+  publishAction = status => (evt) => {
+    const { learningPath, onSelect } = this.props;
     evt.preventDefault();
     if (status.status !== learningPath.status) {
       onSelect(status.action, learningPath);
     }
   };
 
-  const dropDownMenuItemClassName = status => classNames({
-    'dropdown-menu_item': true,
-    active: learningPath.status === status,
-  });
+  dropDownMenuItemsClassName = () =>
+    (
+      classNames({
+        'dropdown-menu_items': true,
+        'dropdown-menu_items_show': this.state.displayDropDown,
+      })
+    );
+
+  dropDownMenuItemClassName = (status) => {
+    const { learningPath } = this.props;
+    return (
+      classNames({
+        'dropdown-menu_item': true,
+        active: learningPath.status === status,
+      })
+    );
+  }
+
+  handleDropDownOnClick = () => {
+    const isShowingMenu = this.state.displayDropDown;
+    this.setState({ displayDropDown: !isShowingMenu });
+  };
+
+  handleDropDownOnMouseEnter = () => {
```

Comment:
kan du ikke lage en funksjon som håndterer både true og false slik at du ikke trenger to funksjoner for dette?
Feks 
```javascript
handleDropDownOnMouseEvents = (displayDropDown) => {
  this.setState({ displayDropDown });
 };
```
og i HTML'en kan du skrive da:
```javascript
<div className="dropdown-menu" onMouseEnter={() => this.handleDropDownOnMouseEvents(true)} onMouseLeave={() => this.handleDropDownOnMouseEvents(false)}>
```